P/pandoc/build_tarballs.jl

```diff
@@ -2,16 +2,16 @@ using BinaryBuilder
 
 # Collection of pre-build pandoc binaries
 name = "pandoc"
-pandoc_ver = "2.19.2"
-version = v"2.19.3"
+pandoc_ver = "3.0.1"
+version = v"3.0.1"
```

Comment:
Now can do this again, to keep things in sync more easily:
```suggestion
version = VersionNumber(pandoc_ver)
```